src/robusta/core/sinks/telegram/telegram_client.py

```
@@ -10,20 +10,24 @@
     def send_message(self, message: str, disable_links_preview: bool = True):
         url = f"{TELEGRAM_BASE_URL}/bot{self.bot_token}/sendMessage"
         message_json = {
             "chat_id": self.chat_id,
             "message_thread_id": self.thread_id,
             "disable_web_page_preview": disable_links_preview,
-            "parse_mode": "Markdown",
             "text": message,
         }
+        if self.parse_mode is not None:
+            message_json["parse_mode"] = self.parse_mode
         response = requests.post(url, json=message_json)
```

Ruff (0.15.17)

[error] Probable use of `requests` call without timeout

(S113)